src/OSPSuite.Core/Domain/ValueOrigin.cs

```diff
@@ -53,6 +98,33 @@ public void UpdateFrom(ValueOrigin valueOrigin)
 
       public string Display => defaultDisplay(this);
 
+      public int CompareTo(ValueOrigin other)
```

Comment:
Needs to implement IComparable and override Equals to allow filtering in UI. 
Because two value origin are the same if they have the same method, source and description, that's what I used to implement comparable. That should be ok as long as we do not do funky stuff with ValueORigins in hash or in dictionary. In that case, we will need to create ValueOriginDTO... but rather not to for such a simple object